recipes/tinyobjloader/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "tinyobjloader" %}
+{% set version = "1.0.6" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 19ee82cd201761954dd833de551edb570e33b320d6027e0d91455faf7cd4c341
+
+build:
+  number: 0
+  noarch: generic
```

Comment:
As the install location for Windows and unix is different this one can't be noarch unfortunately:
```suggestion
```

Reply:
Thanks for taking a look, @chrisburr. Should be good to go now.